docs/configuring-metabase/caching.md

```diff
@@ -24,7 +26,7 @@ These policies determines how long cached results will be stored.
 
 {% include plans-blockquote.html feature="Duration caching policy" %}
 
-Keep the cache for a number of hours. When someone runs a query, Metabase will first check if it's cached the results. If not, it runs the query and caches the results for as long as you set the duration.
+Keep the cache for X number of hours. When someone runs a query, Metabase will first check if it's cached the results. If not, it runs the query and caches the results for as long as you set the duration.
 
 ### Schedule caching policy
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 37]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Maybe "You can schedule the cache to invalidate"?

The bullet points work, I think, but a list with commas might also work well<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/45584?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->